zerver/lib/webhooks/git.py

```diff
@@ -139,7 +140,18 @@ def get_pull_request_event_message(user_name: str, action: str, url: str, number
     else:
         main_message = PULL_REQUEST_OR_ISSUE_MESSAGE_TEMPLATE.format(**kwargs)
 
-    if assignee:
+    if assignees:
+        assignees_string = ""
+        if len(assignees) == 1:
+            assignees_string = "{username}".format(**assignees[0])
+        else:
+            for a in assignees[:-1]:
+                assignees_string += "{username}, ".format(**a)
```

Comment:
see my comment below; I think this part should just be `", ".join(assignees[:-1]) + " and assignees[-1]".